test/acceptance/node_sync_test.go

```diff
@@ -20,19 +21,22 @@ import (
 // (could have mocked the whole thing)
 // Either test with Benchmark Consensus which is makes it easier to generate fake proofs, or use real recorded Lean Helix blocks
 func TestInterNodeBlockSync_WithBenchmarkConsensusBlocks(t *testing.T) {
-
 	NewHarness().
 		WithConsensusAlgos(consensus.CONSENSUS_ALGO_TYPE_BENCHMARK_CONSENSUS).
 		//WithLogFilters(log.ExcludeEntryPoint("BenchmarkConsensus.Tick")).
 		//AllowingErrors().
 		WithSetup(func(ctx context.Context, network *Network) {
+			newRefTime := GenerateNewManagementReferenceTime(0)
```

Comment:
again ... make this committee the default ... in fact i wanted to move all the acceptance and e2e to use a new ctor  of memory provider with a committee (it now does read it from the config ... but you override it each time)